projects/emissary-ingress/emissary/Makefile

```diff
@@ -0,0 +1,49 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=emissary
+REPO_OWNER=emissary-ingress
+
+BINARY_TARGET_FILES=busyambassador capabilities_wrapper example-envoy-metrics-sink k8sregistryctl kat-client kat-server
+SOURCE_PATTERNS=./cmd/busyambassador ./cmd/capabilities_wrapper ./cmd/example-envoy-metrics-sink ./cmd/k8sregistryctl ./cmd/kat-client ./cmd/kat-server
+GO_MOD_PATHS=.
+
+EMISSARY_IMAGE_COMPONENT=emissary-ingress/emissary
+
+IMAGE_NAMES=emissary
+DOCKERFILE_FOLDER=./docker/linux/emissary-image
+
+SIMPLE_CREATE_BINARIES?=false
+
+BASE_IMAGE_REPO?=public.ecr.aws/k1e6s8o8
+BASE_IMAGE_NAME?=eks-distro-minimal-base-python
+
+APPMESH_VERSION:=$(shell cat APPMESH_VERSION)
```

Comment:
mind putting this at the top with the git_tag line.

Reply:
Sure